src/components/ui/UiLink/UiLink.tsx

```diff
@@ -19,10 +20,18 @@ const theme = createTheme({
   },
 });
 
-function UiLink({ children }: { children: React.ReactNode }) {
+function UiLink({
+  children,
+  props,
+}: {
+  children: React.ReactNode;
+  props: any;
```

Comment:
don't use 'any', we use typescript for typing, 'any' breaks typescript role, remove 'any' from everywhere in your code